de.flose.Kochbuch.json

```diff
@@ -0,0 +1,98 @@
+                "strip": true
+            }
+        },
+        {
+            "name": "libgdiplus",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.mono-project.com/sources/libgdiplus/libgdiplus-6.1.tar.gz",
+                    "sha256": "97d5a83d6d6d8f96c27fb7626f4ae11d3b38bc88a1726b4466aeb91451f3255b"
+                }
+            ]
+        },
+        {
+            "name": "kochbuch",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -dm755 /app/opt",
+                "cp -r kochbuch/* /app/opt/",
+                "install -Dm755 kochbuch.sh /app/bin/kochbuch",
+                "touch /app/opt/UpdateDisabled",
+                "install -Dm644 Microsoft.VisualBasic.dll /app/opt/",
+                "install -Dm644 de.flose.Kochbuch.metainfo.xml /app/share/metainfo/de.flose.Kochbuch.metainfo.xml",
+                "install -Dm644 de.flose.Kochbuch.desktop /app/share/applications/de.flose.Kochbuch.desktop",
+                "install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/de.flose.Kochbuch.svg"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.flose.de/download/kochbuch/kochbuch-portable2.6.17.zip",
+                    "dest": "kochbuch",
+                    "sha256": "afe1692ac5ff98166056027ada678441fecf23a03441437957f2fadcf5593138",
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "https://www.flose.de/en/kochbuch/releasenotes",
+                        "version-pattern": "Version ([\\d\\.-]*) Released",
+                        "url-template": "https://www.flose.de/download/kochbuch/kochbuch-portable${version0}.${version1}.${version2}.zip"
+                    }
+                },
+                {
+                    "type": "file",
+                    "path": "kochbuch.sh"
+                },
+                {
+                    "type": "file",
+                    "url": "https://www.flose.de/show/kochbuch/de.flose.Kochbuch.metainfo.xml",
+                    "sha256": "fc1e4640a141c4434c8b34df8fa38030efb5d0efd30d112a558829a613812ba3"
+                },
+                {
+                    "type": "file",
+                    "url": "https://www.flose.de/show/kochbuch/de.flose.Kochbuch.desktop",
+                    "sha256": "fc1617a3bac38ff68ef26b0a1d46838dd341716626099b49acac663b13b55786"
+                },
+                {
+                    "type": "file",
+                    "url": "https://www.flose.de/show/kochbuch/icon.svg",
+                    "sha256": "bb2d36d15ff642cc7d9cdf41d171ff4b78e512802e5fb4058f0322e66b0b8b85"
+                },
+                {
+                    "type": "file",
+                    "url": "https://www.flose.de/download/Microsoft.VisualBasic.dll",
```

Comment:
what's the license for this an why is it downloaded from a third party based on the name....

Reply:
This file is from the Mono project using a MIT license (https://github.com/mono/mono/blob/main/LICENSE)
It's unfortunately not part of org.freedesktop.Sdk.Extension.mono6, so it needs to be downloaded separately.

Reply:
I don't see it there. Maybe it should be built from Mono source? It's sounds fishy to have a random "Microsoft" file downloaded from a rand place on the internet.

Reply:
I've added a mono-basic module to compile the library from source.